### DISCUSSION TITLE
Server updated, but not web and build?

### DISCUSSION THREAD

**Discussion by Deadstarrr** [jellyfin/jellyfin#12461] [Q&A]
I'm new to jellyfin and am a little bit confused. I downloaded the latest version (10.9.9) and installed it on windows, and I see that the server version updated properly, but the web and build versions still both say 10.9.8. Is this normal? Why didn't those update? Thanks!

**Comment by crobibero** [comment]
Typically it is a browser caching issue. Try in a private browser
